src/hooks/useDragAndDrop.ts

```diff
@@ -23,9 +25,17 @@ type DragAndDropOptions = {
 /**
  * @param dropZone – ref to the dropZone component
  */
-export default function useDragAndDrop({dropZone, onDrop = () => {}, shouldAllowDrop = true, isDisabled = false, shouldAcceptDrop = () => true}: DragAndDropParams): DragAndDropOptions {
+export default function useDragAndDrop({
+    dropZone,
+    onDrop = () => {},
+    shouldAllowDrop = true,
+    isDisabled = false,
+    shouldAcceptDrop = () => true,
+    shouldClosePopover = true,
```

Comment:
Do we ever set this to `false`?

Reply:
Then I'd say to remove this parameter in the YAGNI style. It can be easily added when needed.